src/__tests__/hooks/useOnClickOutside.test.js

```diff
@@ -56,9 +52,14 @@ describe('useOnClickOutside', () => {
       bubbles: true,
       cancelable: true,
     });
-    unmount();
+
     outsideElement.dispatchEvent(event);
+    expect(handler).toHaveBeenCalledTimes(1);
 
-    expect(handler).not.toHaveBeenCalled();
+    unmount();
+
+    outsideElement.dispatchEvent(event);
+    expect(handler).toHaveBeenCalledTimes(1);
+    expect(handler).not.toHaveBeenCalledTimes(2);
```

Comment:
nitpick: if you've already checked it was called once, the second assert is redundant